Sources/XcodeProj/Project/XcodeProj.swift

```diff
@@ -130,6 +130,12 @@ extension XcodeProj: Writable {
     /// - Parameter override: if workspace should be overridden. Default is true.
     ///   If false will throw error if workspace already exists at the given path.
     public func writeWorkspace(path: Path, override: Bool = true) throws {
+
+        // prevent overwriting identical workspace data
+        if let prevData = try? XCWorkspaceData(path: path), prevData == workspace.data {
```

Comment:
I'd rewrite this into a more readable and concise version (IMHO)
```swift
guard workspace.data != try? XCWorkspaceData(path) else { return } 
```

Reply:
Good point, that reads so much better! Done ✅

Reply:
I'd also modify the comment that describes why overwriting the data is bad (something like it forces the Xcode project to be reloaded)